api/HTMLFieldSetElement.json

```diff
@@ -70,10 +70,10 @@
               "version_added": true
             },
             "opera": {
-              "version_added": true
+              "version_added": "≤12.1"
```

Comment:
https://github.com/mdn/browser-compat-data/pull/7054 doesn't update `disabled` (this entry) or the following entries for `elements`, `validationMessage` or `validity` updated in this PR.

Is that because they *weren't* introduced in Chrome 1? If they were not, then when reviewing I'd check if they were introduced so late that they were actually removed in Opera 15.

Reply:
The collector data that went into this PR _did_ include Opera 15 in its results; if it wasn't present in Opera 15, the script would generate a `version_removed` statement for us.

Most likely this is another case of a feature being in Chrome 1 but the existing data says otherwise -- I promise to follow up with some more PRs to fix those issues!

Reply:
Super, thank you!